crates/spidermonkey-embedding-splicer/src/stub_wasi.rs

```diff
@@ -82,7 +100,9 @@ pub fn stub_wasi(wasm: Vec<u8>, features: Vec<Features>) -> Result<Vec<u8>> {
         stub_io(&mut module)?;
```

Comment:
We probably need to extend `has_io` to also include `|| module_imports.contains("wasi:filesystem") || module_imports.contains("wasi:sockets")`.